lib/govuk_app_config/govuk_content_security_policy.rb

```diff
@@ -91,6 +90,16 @@ def self.build_policy(policy)
   def self.configure
     Rails.application.config.content_security_policy_report_only = ENV.include?("GOVUK_CSP_REPORT_ONLY")
 
+    # Sets a nonce per request that is set in the CSP directives for script_src and
+    # style_src.
+    # Note: if an applications needs to set unsafe-inline they will need to unset this nonce too (by setting
+    # this config option to nil)
+    Rails.application.config.content_security_policy_nonce_generator = ->(_request) { SecureRandom.base64(16) }
+
+    # This only applies the nonce generator to the script-src directive. We need this to
+    # use unsafe-inline for style-src as a nonce will override it.
+    Rails.application.config.content_security_policy_nonce_directives = %w[script-src]
```

Comment:
In other words, once we can apply unsafe-inline to the style dir, we can remove this config line (as I assume it would apply to both by default)?
Or would we be adding style-src to this array?
Would be good to make the comment clearer about what will happen in future.